sysl2/sysl/seqs.go

```diff
@@ -0,0 +1,288 @@
+			}
+		}
+	} else {
+		if endpoints == nil {
+			return
+		}
+		spep := constructSimpleParser("", endpoint_format)
+		spapp := constructSimpleParser("", app_format)
+		sd := &sequenceDiagParam{
+			endpoints:   endpoints,
+			AppLabeler:  spapp,
+			EndpointLabeler: spep,
+			title:       title,
+			blackboxes:  blackboxes,
+		}
+		out, _ := generateSequenceDiag(mod, sd)
+		seqs.OutputPlantuml(output, plantuml, out)
+	}
+}
+
+// DoGenerateSequenceDiagrams generate sequence diagrams for the given model
+func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, flags *flag.FlagSet, args []string) int {
+	var endpoints_flag, apps_flag, blackboxes_flag, modules_flag arrayFlags
+	root_model := flags.String("root-model", ".", "sysl root directory for input model file (default: .)")
+	endpoint_format := flags.String("endpoint_format", "%(epname)", "Specify the format string for sequence diagram endpoints. "+
+		"May include %%(epname), %%(eplongname) and %%(@foo) for attribute foo(default: %(epname))")
+	app_format := flags.String("app_format", "%(appname)", "Specify the format string for sequence diagram participants. "+
+		"May include %%(appname) and %%(@foo) for attribute foo(default: %(appname))")
+	title := flags.String("title", "", "diagram title")
+	plantuml := flags.String("plantuml", "", strings.Join([]string{"base url of plantuml server",
+		"(default: $SYSL_PLANTUML or http://localhost:8080/plantuml",
+		"see http://plantuml.com/server.html#install for more info)"}, "\n"))
+	output := flags.String("output", "%(epname).png", "output file(default: %(epname).png)")
+	flags.Var(&endpoints_flag, "endpoint", "Include endpoint in sequence diagram")
+	flags.Var(&apps_flag, "app", "Include all endpoints for app in sequence diagram (currently "+
+		"only works with templated --output). Use SYSL_SD_FILTERS env (a "+
+		"comma-list of shell globs) to limit the diagrams generated")
+	flags.Var(&blackboxes_flag, "blackbox", "Apps to be treated as black boxes")
+	flags.Var(&modules_flag, "modules", strings.Join([]string{"input files without .sysl extension and with leading /",
+		"eg: /project_dir/my_models",
+		"combine with --root if needed"}, "\n"))
+
+	// Following variables currently are unused. Keep them to align with the python version.
+	filter := flags.String("filter", "", "Only generate diagrams whose output paths match a pattern")
+	no_activations := flags.Bool("no-activations", true, "Suppress sequence diagram activation bars(default: true)")
+	verbose := flags.Bool("verbose", false, "Report each output(default: false)")
+	expire_cache := flags.Bool("expire-cache", false, "Expire cache entries to force checking against real destination(default: false)")
+	dry_run := flags.Bool("dry-run", false, "Don't perform confluence uploads, but show what would have happened(default: false)")
+
+	err := flags.Parse(args[1:])
+	if err != nil {
+		log.Errorf("arguments parse error: %v", err)
+	}
+	log.Warnf("root_model: %s\n", *root_model)
```

Comment:
Use `Debugf`.

Reply:
It should cut down the log level to debug. Fix it